docs/data-sources/cache.md

```diff
@@ -0,0 +1,99 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "encore_cache Data Source - terraform-provider-encore"
+subcategory: ""
+description: |-
+  Encore provisioned cache information
+---
+
+# encore_cache (Data Source)
+
+Encore provisioned cache information
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The name of the Encore resource
+
+### Optional
+
+- `env` (String) The environment of the Encore resource. Defaults to the provider environment
+
+### Read-Only
+
+- `aws_redis` (Attributes) (see [below for nested schema](#nestedatt--aws_redis))
+- `database_number` (Number)
+- `gcp_redis` (Attributes) (see [below for nested schema](#nestedatt--gcp_redis))
+
+<a id="nestedatt--aws_redis"></a>
+### Nested Schema for `aws_redis`
+
+Read-Only:
+
+- `arn` (String)
+- `parameter_group` (Attributes) (see [below for nested schema](#nestedatt--aws_redis--parameter_group))
+- `security_group` (Attributes) (see [below for nested schema](#nestedatt--aws_redis--security_group))
+- `subnet_group` (Attributes) (see [below for nested schema](#nestedatt--aws_redis--subnet_group))
```

Comment:
I guess all subnets must belong to the same VPC? Would be nice to expose the vpc directly here on the top level if so, so you don't need to do the extra indirection via `subnet_group.subnets[0]`

Edit: I guess that's maybe not true if we have replication across regions?